glossary/affiliate-definition.md

```diff
@@ -0,0 +1,7 @@
+---
+type: Resource
+---
+
+# Affiliate
+
+Non-profit, non-governmental organization that has signed a letter of interest in collaborating with the Foundation for Public Code.
```

Comment:
I would suggest removing `non-govermental` as some might be, e.g. we have a relationship with OASC, a network of cities. The requirement for the kind of relationship can also be looser, with OS2 we have a _memorandum of understanding_ and with OW2 an _affiliate contract_.

```suggestion
A non-profit organization that has an aligned mission with us and we have formal relations with.
```

Reply:
Ah yes, and the 'aligned with our mission' seems important :)